lib/utils/pyexec.c

```diff
@@ -548,8 +548,8 @@ int pyexec_friendly_repl(void) {
 
     #if defined(USE_HOST_MODE) && MICROPY_HW_HAS_LCD
     // in host mode, we enable the LCD for the repl
-    mp_obj_t lcd_o = mp_call_function_0(mp_load_name(qstr_from_str("LCD")));
-    mp_call_function_1(mp_load_attr(lcd_o, qstr_from_str("light")), mp_const_true);
+    mp_obj_t lcd_o = mp_call_function_0(mp_load_name(MP_QSTR_LCD));
+    mp_call_function_1(mp_load_attr(lcd_o, MP_QSTR_light), mp_const_true);
```

Comment:
I'd be happy to just get rid of this block of code.  It probably doesn't work anymore, and is highly specific to stm32.

Reply:
@dhylands I think you wrote this, any comments? Looking at other usaes of `USE_HOST_MODE` this does look deprecated indeed. Otherwsie it should rather be moved to a port-specific initialization?

Reply:
I think I wrote this, see 75abee206d1a575aa98a486d043c94d64df432c1

Reply:
Right, didn't look further than pyexec's git history. Removed it, but you might want to adjust the commit message for clarity.